src/arm/96boards.c

```diff
@@ -97,7 +105,12 @@ mraa_board_t* mraa_96boards()
             ls_gpio_pins = hikey_ls_gpio_pins;
             b->uart_dev[0].device_path = hikey_serialdev[0];
             b->uart_dev[1].device_path = hikey_serialdev[1];
-	}
+	     } else if (mraa_file_contains(DT_BASE "/model", "s900")) {
```

Comment:
This should be on a separate line (hint: run `clang-format` on your changes using config file from the root of the repo)